construdelas/Desafio_dev_jr/desafio_dev_jr.py

```diff
@@ -0,0 +1,85 @@
+import pandas as pd
```

Comment:
[suggestion] O que você acha de colocar uma docstring no início do arquivo explicando como este módulo funciona?